pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>atlassian-bitbucket-server-integration</artifactId>
-    <version>4.1.1-SNAPSHOT</version>
+    <version>4.1.2-SNAPSHOT</version>
```

Comment:
Will bump this to 4.1.3-SNAPSHOT, this PR includes the changes when i manually released 4.1.1